lib/chef/provider/aws_rds_instance.rb

```diff
@@ -0,0 +1,49 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSProvider
+  REQUIRED_OPTIONS = %w(db_instance_identifier allocated_storage engine
+                        db_instance_class master_username master_user_password)
+
+  OTHER_OPTIONS = %w(engine_version multi_az iops publicly_accessible db_name db_port)
+
+  def update_aws_object(instance)
+    Chef::Log.warn("aws_rds_instance does not support modifying a started instance")
+  end
+
+  def create_aws_object
+    converge_by "Create new RDS instance #{new_resource.db_instance_identifier} in #{region}" do
+      new_resource.driver.rds.client.create_db_instance(create_options)
+    end
+  end
+
+  def destroy_aws_object(instance)
+    converge_by "Deleting RDS instance #{new_resource.db_instance_identifier} in #{region}" do
+      instance.delete(skip_final_snapshot: true)
+    end
+  end
+
+  def create_options
+    opts = {}
+    opts.merge(new_resource.additional_options)
```

Comment:
As tested for, the explicit value overrides `additional_options` (see [test](https://github.com/chef/chef-provisioning-aws/blob/08e5c2d3eed5f93550973167a6fce554e3a8ab38/spec/integration/aws_rds_instance_spec.rb#L69-L74)).

I'll add some documentation.